Documentation/ApiOverview/CachingFramework/Index.rst

```diff
@@ -12,41 +12,51 @@ Caching in TYPO3
 ----------------
 
 TYPO3 uses multiple caching strategies to ensure fast content delivery. Depending
-on the content a page contains, TYPO3 chooses the best caching strategy for that use case.
+on the content a page contains, TYPO3 chooses the best strategy for the usage of caches.
+Each page has its own cache entry. There is no duplication of a cache between different pages. 
```

Comment:
I think we should distinguish here whether this is supposed to be about "cache_pages" or about caching in TYPO3 in general. There may well be duplication of cache entries for elements on a certain page if we are not talking about cache_pages specifically.

Reply:
If you know something about a cache duplication of content elements on a page, then you are welcome to tell us about it. 
I only know that TYPO3 caches the pages by itself. And each page cache already contains the content caches. I have never heard about a special content element cache.

Reply:
see https://b13.com/blog/caching-in-typo3-part-3 for an in-depth explanation - especially point 4.

Reply:
This documentation only describes the 
"4. What caches exist in TYPO3 by default?" 
And from here only the "Frontend-related caches" the “pages cache".

Reply:
Hu? But we are here in the TYPO3 Caching Framework general documentation, not in a specific one about pages cache specifically if I see that correctly, so only explicitly describing one of the caches is imho more confusing than not describing any of them as it would give the impression, that the pages cache is the only/most important one, while ignoring things like menu caches or content element section caches (especially because this documentation refers to the possibility to set cache options via typoscript which will then result in entries in the cache hash table and not in cache_pages).

Reply:
Considering the different caches it's really confusing that no overview is given here but directly the page-cache mentioned.  
Better would be a list with the different caches.  
If the text-style shall be kept it should be on base of a common page where each kind of cache is shortly explained related to the corresponding page-element where the page-cache itself could be the first one but perhaps better the last one to focus a bit more on the other caches like menu-cache etc.

Reply:
I am sorry, but I am only using the TYPO3 page cache so far. I have not enough information about the other caches which there are.